main.go

```diff
@@ -137,13 +137,19 @@ func main() {
 	}
 	queueUrls := strings.Split(queueVar, ",")
 
+	listenVar, listenSet := os.LookupEnv("LISTEN")
+
+	if !listenSet || listenVar == "" {
+		listenVar = ":8080"
```

Comment:
I'd suggest templating the ":" so that the value of the variable can just be the port number. Otherwise it would be easy to make a mistake and set the variable to `80` instead of `:80`.